tests/unit/lms/views/predicates/_lti_launch_test.py

```diff
@@ -186,6 +199,13 @@ def pyramid_request(self, pyramid_request):
         }
         return pyramid_request
 
+    @pytest.fixture
```

Comment:
Make sure this applies to all the tests in this class (not just the ones that explicitly use the `assignment_service` fixture):

```suggestion
    @pytest.fixture(autouse=True)
```

You could also make this fixture a module-level function instead of a class method, then it'd apply to all tests in this module. (Would require some small tweaks to `TestDBConfigured`).